gst/tensor_reposrc/tensor_reposrc.c

```diff
@@ -176,6 +178,17 @@ gst_tensor_reposrc_getcaps (GstBaseSrc * src, GstCaps * filter)
       cap = gst_caps_new_any ();
   }
 
+  check = gst_caps_from_string (CAPS_STRING);
+
+  if (!gst_caps_intersect (cap, check)) {
```

Comment:
The return of gst_caps_intersect needs to be unref'ed before exit.

Reply:
done. thanks.